GCMessengerSDKSample/Podfile.lock

```diff
@@ -89,23 +89,23 @@ SPEC REPOS:
     - GenesysCloudMessenger
 
 SPEC CHECKSUMS:
-  FirebaseCore: a282032ae9295c795714ded2ec9c522fc237f8da
-  FirebaseCoreExtension: cda74ddfb001224bd8fd1d6e74698b4ed07803de
-  FirebaseCoreInternal: 0c569513412da9f3b31bd0b340013bbee8f295c5
-  FirebaseCrashlytics: cfc69af5b53565dc6a5e563788809b5778ac4eac
-  FirebaseInstallations: 771177d89d6c451dc6e50085ec82e2fc77ed0a4a
-  FirebaseRemoteConfigInterop: 477b26fdeb8fb5fbaf22fa9db5343b42289dc7db
-  FirebaseSessions: adcec8b72d0066a385e3affcd1bcb1ebb3908ce6
-  GenesysCloud: 5b3d7511020885bf401d7e3fc53fcc67e42c0424
-  GenesysCloudCore: 8cb6d0255c8c0ddec19801e19362cf7a59780e9f
-  GenesysCloudMessenger: 6e3e86fae6fbbdc322178a706e7dfe7d10ff8a03
+  FirebaseCore: 8542de610f35f86196ba26cdb2544565a5157c8e
+  FirebaseCoreExtension: 30bb063476ef66cd46925243d64ad8b2c8ac3264
+  FirebaseCoreInternal: ac26d09a70c730e497936430af4e60fb0c68ec4e
+  FirebaseCrashlytics: ba7b6a55dc10393f6583d87d8600d0d3ab2671d8
+  FirebaseInstallations: 58cf94dabf1e2bb2fa87725a9be5c2249171cda0
+  FirebaseRemoteConfigInterop: c3a5c31b3c22079f41ba1dc645df889d9ce38cb9
+  FirebaseSessions: 655ff17f3cc1a635cbdc2d69b953878001f9e25b
+  GenesysCloud: c9f690ef26b23401ca2a549a588413f8e6c02ad4
+  GenesysCloudCore: aa849f1d2a2975dc565f305d8dc8a398ddaa1898
+  GenesysCloudMessenger: 2ec0d98abb31f21698f2f98f2b65f4992de02860
   GenesysCloudMessengerTransport: d0ef0e3a858b1928354327185169f66f4fa38422
   GoogleDataTransport: aae35b7ea0c09004c3797d53c8c41f66f219d6a7
   GoogleUtilities: 26a3abef001b6533cf678d3eb38fd3f614b7872d
   nanopb: fad817b59e0457d11a5dfbde799381cd727c1275
   PromisesObjC: f5707f49cb48b9636751c5b2e7d227e43fba9f47
   PromisesSwift: 9d77319bbe72ebf6d872900551f7eeba9bce2851
 
-PODFILE CHECKSUM: e14824ed95d415d742c049ab0d943f3fef2f2cf2
+PODFILE CHECKSUM: 6eabfb6ed6493e6b2670f950c2fdbe8d97d65407
 
-COCOAPODS: 1.15.2
```

Comment:
Is this version downgrade necessary or is it by accident?

Reply:
Updated